ir/memory.cpp

```diff
@@ -1454,7 +1456,8 @@ unsigned Memory::getStoreByteSize(const Type &ty) {
   if (ty.isPtrType())
     return divide_up(bits_program_pointer, 8);
 
-  if (auto aty = ty.getAsAggregateType()) {
+  auto aty = ty.getAsAggregateType();
+  if (aty && !isIntVector(ty)) {
```

Comment:
why only int? we only distinguish between ptr and non-ptr in memory.

Reply:
Rename `isIntVector` to `isNonPtrVector` and expand its coverage too